content/swiftui/concepts/gestures/gestures.md

```diff
@@ -0,0 +1,99 @@
+---
+Title: 'Gestures' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Defines interactions from taps, clicks, and swipes to fine-grained gestures.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'iOS'
+  - 'Mobile Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'SwiftUI ViewModifiers'
+  - 'SwiftUI'
+  - 'iOS'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-iOS-course'
+  - 'path/iOS-path'
+---
+
+Views can respond to taps, drags, pinches and other gestures just by adding **Gesture** modifiers to your View.
+
+## Tap Gestures
+
+This is the most basic type of Gesture.
+The `.onTapGesture()` modifier can be called on the view desired to track the gesture on.
+
+```Swift
+Text("Hello, World!") // SwiftUI Text view.
+            .onTapGesture { // Using this modifier, a specific action can be run when the View is tapped.
+
+                print("View Tapped") // This function will print "View Tapped" in the console.
+
+            }
+
+```
+
+This example will output the following when the view is tapped:
+
+![Screen_Recording_2023-05-20_at_4_05_58_PM_AdobeExpress](https://github.com/EngOmarAhmedIOS/docs/media/Screen_Recording_2023-05-20_at_4_05_58_PM_AdobeExpress.gif)

```

Comment:
@EngOmarAhmedIOS two things about this link. First, rename the image something more meaningful like `swiftui-gestures-example.gif`. Second, the URL you need to use is `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name. That path won't render until this is merged, but we want it pointing to the main Docs media folder.

Reply:
I did as you told me, I hope I did it right this time 😅 @SSwiniarski